### PR TITLE
[bug 1137235] Button styling and profile options

### PR DESCRIPTION
**Pull request by Rik**
- Removed unnecessary styles in buddyup.css
- Styled the options in the profile page
- Wording changes on profile page
- Changed the way we store the handset type
- Changed some <buttons> to <a> because of building blocks
- Includes a temporary fix for buttons on the question page

### CODE REVIEW

File: app/css/profile.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.QuestionFilters {
+    margin: 15px;
+}
+
+.QuestionFilters-label {
+    display: block;
```
Comment (rehandalal) [inline app/css/profile.css]: alphabetize

File: app/css/profile.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    margin: 15px 15px 10px;
+    font-size: 18px;
+    line-height: 22px;
+}
+
+.QuestionFilters-description {
```
Comment (rehandalal) [inline app/css/profile.css]: alphabetize

File: app/views/thread_header.html [hunk 3]
```diff
@@ -6,11 +6,19 @@ <h3 class="question-cat">
...
-      {% if selected_category == key %}selected{% endif %}>
-      {{ value }}
-    </option>
-  {% endfor %}
-</select>
+
+{# This <ul><li> is needed to get the right Building block styling for that button #}
+<ul>
+  <li>
+    <div class="button" data-icon="forward">
```
Comment (rehandalal) [inline app/views/thread_header.html]: i believe this should be `data-icon="forward-light"`

Comment (rehandalal) [inline app/views/thread_header.html]: The icon on this button is italicized. It should not be.

Comment (rehandalal) [inline app/views/thread_header.html]: also the icons on all the buttons are too small

Comment (Rik) [inline app/views/thread_header.html]: The size is set by the building blocks to 1.6rem, which is 16px for us so it should be the proper size per building block styling.
Also, the examples on http://buildingfirefoxos.com/building-blocks/buttons.html use forward and not forward-light. So I'll just fix the italic issue for now.

File: app/css/icons.css [hunk 4]
```diff
@@ -9,12 +9,14 @@
     font-family: "gaia-icons";
     content: attr(data-icon);
     text-rendering: optimizeLegibility;
+    font-style: normal;
```
Comment (rehandalal) [inline app/css/icons.css]: nit: alphabetize

File: app/css/icons.css [hunk 5]
```diff
@@ -9,12 +9,14 @@
...
 }
 [data-icon-after]:after {
     float: right;
     font-family: "gaia-icons";
     content: attr(data-icon-after);
     text-rendering: optimizeLegibility;
+    font-style: normal;
```
Comment (rehandalal) [inline app/css/icons.css]: nit: alphabetize

**Pull request merged**